doc/astspec.txt

```diff
@@ -10,7 +10,6 @@ contains:
 
   type
     NimNodeKind = enum     ## kind of a node; only explanatory
-      nnkNone,             ## invalid node kind
       nnkEmpty,            ## empty node
```

Comment:
For the sake of accuracy, should `nnkError` also be listed as the first node here?

Reply:
Thanks, I missed that as my first approach was replacing nkNone with nkEmpty .
Addressed in https://github.com/nim-works/nimskull/pull/1229/commits/d7c23d9622be02244a018f40d2261fe7091b0dc2 .